<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWindowCourse</class>
 <widget class="QMainWindow" name="QWindowCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EduDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">.QWidget#centralWidget{
	border: 1px solid #262626;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_main" native="true">
      <property name="minimumSize">
       <size>
        <width>1220</width>
        <height>786</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1220</width>
        <height>786</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_main{
  border: none;
  background-color: #ffffff;
}   </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_top" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#widget_top{
	background-color: #262626;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="minimumSize">
             <size>
              <width>154</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>154</width>
              <height>23</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/Course/course/logo.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_course_name">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Microsoft YaHei&quot;;
color: #bbbbbb;</string>
            </property>
            <property name="text">
             <string>英语口语初级(ID:1237281)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_user_name">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Microsoft YaHei&quot;;
color: #bbbbbb;</string>
            </property>
            <property name="text">
             <string>老师</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	font: 10pt &quot;Microsoft YaHei&quot;;
	color: #bbbbbb;
	background-color: #00262626;
}</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>horizontalSpacer_6</zorder>
         <zorder>label_14</zorder>
         <zorder>btn_close</zorder>
         <zorder>label_user_name</zorder>
         <zorder>label_course_name</zorder>
         <zorder>horizontalSpacer</zorder>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_bottom" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_member" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>736</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>736</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #333333;
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_main_share" native="true">
            <property name="styleSheet">
             <string notr="true">#widget_main_share{
	background-color: #ffffff;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_message" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>736</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>736</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #f2f2f2;</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/Course.qrc"/>
 </resources>
 <connections/>
</ui>
